### PR TITLE
add full DELPHI author lists (merge of all papers)

### PR DESCRIPTION
**Pull request by schwicke**

### CODE REVIEW

File: cernopendata/modules/fixtures/data/records/delphi-author-list-2000.json [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+      ]
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "note": {
+      "description": "This is a list of DELPHI Collaboration authors in March 2000, including their affiliations."
+    },
+    "publisher": "CERN Open Data Portal",
+    "title": "DELPHI author list",
```
Comment (psaiz) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-2000.json:535]: Missing recid?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 2]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Di~Diodato, A.",
+          "affiliation": "Dipartimento di Fisica, Universita di Roma II; INFN, Tor Vergata, IT-00173 Rome, Italy"
+        },
+        {
+          "name": "Dijkstra, H.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland"
+        },
+        {
+          "name": "Di~Simone, A.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Please replace `~` by an empty space.

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 3]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Adami, F.",
+          "affiliation": "DAPNIA - Service de Physique des Particules, CEA-Saclay, FR-91191 Gif-sur-Yvette Cedex, France"
+        },
+        {
+          "name": "Adam, W.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institut fuer Hochenergiephysik, Oesterr. Akad. d. Wissensch., Nikolsdorfergasse 18, AT-1050 Vienna, Austria"
+        },
+        {
+          "name": "A.Della~Riccia, B.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Is `A.Della~Riccia, B.` name recognised correctly? Should perhaps be `Della Riccia, B A`?

(judging from https://inspirehep.net/authors/2571661)

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 4]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Nygren, A.",
+          "affiliation": "Department of Physics, University of Lund, Solvegatan 14, SE-223 63 Lund, Sweden"
+        },
+        {
+          "name": "Oberschulte-Beckmann, W.",
+          "affiliation": "Institut fuer Experimentelle Kernphysik, Universitaet Karlsruhe, Postfach 6980, DE-76128 Karlsruhe, Germany"
+        },
+        {
+          "name": "Oberschulte~gen.~Beckmann, W.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Oberschulte~gen.~Beckmann, W.` and `Oberschulte-Beckmann, W.` are probably the same person?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 5]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Palacios, J.P.",
+          "affiliation": "Department of Physics, University of Liverpool, P.O. Box 147, Liverpool L69 3BX, UK"
+        },
+        {
+          "name": "Palka, H.",
+          "affiliation": "Institute of Nuclear Physics and University of Mining and Metalurgy, Ul. Kawiory 26a, PL-30055 Krakow, Poland;KRAKOW1; NIKHEF, Postbus 41882, NL-1009 DB Amsterdam, The Netherlands"
+        },
+        {
+          "name": "P.Allport, P.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Probably `Allport, P P`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 6]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Ajinenko, I.",
+          "affiliation": "Inst. for High Energy Physics, Serpukov P.O. Box 35, Protvino, (Moscow Region), Russian Federation"
+        },
+        {
+          "name": "Akesson, T.",
+          "affiliation": "Department of Physics, University of Lund, Solvegatan 14, SE-223 63 Lund, Sweden"
+        },
+        {
+          "name": "A.Khomenko, B.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Probably `Khomenko, A B`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 7]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Asman, B.",
+          "affiliation": "Fysikum, Stockholm University, Box 6730, SE-113 85 Stockholm, Sweden"
+        },
+        {
+          "name": "Astier, P.",
+          "affiliation": "LPNHE, IN2P3-CNRS, Univ.~Paris VI et VII, Tour 33 (RdC), 4 place Jussieu, FR-75252 Paris Cedex 05, France"
+        },
+        {
+          "name": "A.Tyapkin, I.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Probably `Tyapkin, I A`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 8]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Brenner, R.A.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Helsinki Institute of Physics, HIP, P.O. Box 9, FI-00014 Helsinki, Finland; Department of Radiation Sciences, University of Uppsala, P.O. Box 535, SE-751 21 Uppsala, Sweden"
+        },
+        {
+          "name": "Brenner, R.",
+          "affiliation": "Department of Radiation Sciences, University of Uppsala, P.O. Box 535, SE-751 21 Uppsala, Sweden"
+        },
+        {
+          "name": "B.Renton, P.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Probably `Renton, P B`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 9]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Haug, S.",
+          "affiliation": "Physics Department, University of Oslo, Blindern, NO-1000 Oslo 3, Norway"
+        },
+        {
+          "name": "Hauler, F.",
+          "affiliation": "Institut fuer Experimentelle Kernphysik, Universitaet Karlsruhe, Postfach 6980, DE-76128 Karlsruhe, Germany"
+        },
+        {
+          "name": "H.Crawley, B.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Probably `Crawley, B H`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 10]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Filippas, T.A.",
+          "affiliation": "N Dipartimento di Fisica, Universit`a di Trieste; INFN, Via A. Valerio 2, IT-34127 Trieste, Italy; Istituto di Fisica, Universit`a di Udine, IT-33100 Udine, Italy- Physics Laboratory, University of Athens, Solonos Str. 104, GR-10680 Athens, Greece"
+        },
+        {
+          "name": "Firestone, A.",
+          "affiliation": "Department of Physics and Astronomy, Iowa State University, Ames IA 50011-3160, USA"
+        },
+        {
+          "name": "Fischer, P.-A.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Fischer, P-A` or `Fischer P A` depending on the importance of hyphen. (here and elsewhere in the PR)

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 11]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Jean-Marie, B.",
+          "affiliation": "Universite de Paris-Sud, Lab. de l'Accelerateur Lineaire, IN2P3-CNRS, Bat. 200, FR-91405 Orsay Cedex, France"
+        },
+        {
+          "name": "Jeans, D.",
+          "affiliation": "Department of Physics, University of Oxford, Keble Road, Oxford OX1 3RH, UK"
+        },
+        {
+          "name": "Johansen, Kristine~M.E.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Perhaps `Johansen, K M E` if Kristine is the first name?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 12]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Korcyl, K.",
+          "affiliation": "Institute of Nuclear Physics and University of Mining and Metalurgy, Ul. Kawiory 26a, PL-30055 Krakow, Poland"
+        },
+        {
+          "name": "Korytov, A.V.",
+          "affiliation": "Joint Institute for Nuclear Research, Dubna, Head Post Office, P.O. Box 79, RU-101 000 Moscow, Russian Federation"
+        },
+        {
+          "name": ", K.Osterberg.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Probably `Osterberg, K`?  Note the leading whitespace; perhaps family name was ill recognised.

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 13]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Mazzucato, F.",
+          "affiliation": "Dipartimento di Fisica, Universita di Padova; INFN, Via Marzolo 8, IT-35131 Padua, Italy;GENEVA"
+        },
+        {
+          "name": "Mazzucato, M.",
+          "affiliation": "Dipartimento di Fisica, Universita di Padova; INFN, Via Marzolo 8, IT-35131 Padua, Italy"
+        },
+        {
+          "name": "M.Benlloch, J.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Benlloch, J M`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 14]
```diff
@@ -0,0 +1,4427 @@
...
+          "affiliation": "VALE IPNP of MFF, Charles Univ., Areal MFF, V Holesovickach 2, CZ-180 00, Praha 8, Czech RepublicIA"
+        },
+        {
+          "name": "M.Brunet, J.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Brunet, J M`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 15]
```diff
@@ -0,0 +1,4427 @@
...
+          "affiliation": "College de France, Lab. de Physique Corpusculaire, IN2P3-CNRS, FR-75231 Paris Cedex 05, France"
+        },
+        {
+          "name": "Mc~Cubbin, M.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: Please replace tilde here and elsewhere by an empty space.

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 16]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Lamsa, J.W.",
+          "affiliation": "Department of Physics and Astronomy, Iowa State University, Ames IA 50011-3160, USA"
+        },
+        {
+          "name": "Lanceri, L.",
+          "affiliation": "Dipartimento di Fisica, Universit`a di Trieste; INFN, Via A. Valerio 2, IT-34127 Trieste, Italy; Istituto di Fisica, Universit`a di Udine, IT-33100 Udine, Italy"
+        },
+        {
+          "name": "L.Andrieux, M.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Andrieux, M L`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 17]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Lounis, A.",
+          "affiliation": "Institut de Recherches Subatomiques, IN2P3 - CNRS-ULP - BP20, FR-67037 Strasbourg Cedex, France"
+        },
+        {
+          "name": "Lozano, J.J.",
+          "affiliation": "VALE IPNP of MFF, Charles Univ., Areal MFF, V Holesovickach 2, CZ-180 00, Praha 8, Czech RepublicIA"
+        },
+        {
+          "name": "L.Read, A.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Read, A L`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 18]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Tyndel, M.",
+          "affiliation": "Rutherford Appleton Laboratory, Chilton, Didcot OX11 OQX, UK"
+        },
+        {
+          "name": "Tzamarias, S.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institute of Nuclear Physics, N.C.S.R. Demokritos, P.O. Box 60228, GR-15310 Athens, Greece; Department of Physics, University of Liverpool, P.O. Box 147, Liverpool L69 3BX, UK"
+        },
+        {
+          "name": "U.Chung, S.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Chung, S U`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 19]
```diff
@@ -0,0 +1,4427 @@
...
+        {
+          "name": "Winter, M.",
+          "affiliation": "Institut de Recherches Subatomiques, IN2P3 - CNRS-ULP - BP20, FR-67037 Strasbourg Cedex, France"
+        },
+        {
+          "name": "Witek, M.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institute of Nuclear Physics and University of Mining and Metalurgy, Ul. Kawiory 26a, PL-30055 Krakow, Poland;KRAKOW1"
+        },
+        {
+          "name": "W.Lane, D.",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json]: `Lane, D W`?

File: cernopendata/modules/fixtures/data/records/delphi-author-list-all.json [hunk 20]
```diff
@@ -0,0 +1,4237 @@
...
+          "affiliation": "DAPNIA - Service de Physique des Particules, CEA-Saclay, FR-91191 Gif-sur-Yvette Cedex, France",
+          "name": "Adami, F"
+        },
+        {
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institut fuer Hochenergiephysik, Oesterr. Akad. d. Wissensch., Nikolsdorfergasse 18, AT-1050 Vienna, Austria",
+          "name": "Adam, W"
+        },
+        {
+          "affiliation": "Dipartimento di Fisica Sperimentale, Universita di Torino; INFN, Via P. Giuria 1, IT-10125 Turin, Italy",
+          "name": "Della Riccia, B A"
```
Comment (psaiz) [inline cernopendata/modules/fixtures/data/records/delphi-author-list-all.json:23]: After the last fixes, the order of the authors is not in alphabetical order. Should that be fixed?

**Review by tiborsimko — APPROVED** [review]

**Pull request merged**